README.md

```diff
@@ -242,6 +242,23 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+## Add pre-commit plugin
```

Comment:
I try it on the repo itself with this branch as rev, but I wasn't able to make it work :(
Also I got some linting errors

Reply:
I changed the readme, maybe this is the step mising in you test : 
https://github.com/lumapps/commit-message-validator/blame/chore/precommit_hook/README.md#L266

Reply:
Actually witout `--hook-type commit-msg` it can't work. I fixed the docs